pylab/website/views.py

```diff
@@ -71,7 +71,26 @@ def about(request):
 
 def event_details(request, year, month, day, slug):
     event = get_object_or_404(Event, starts__year=year, starts__month=month, starts__day=day, slug=slug)
-    return render(request, 'website/event_details.html', {'event': event})
+    attendance = Attendance.objects.filter(event=event)
+    if request.method == 'POST':
+        form = website_forms.AttendanceForm(request.POST)
+        if form.is_valid():
+            instance = Attendance.objects.get(event=event, attendee=request.user)
+            instance.response = form.cleaned_data['response']
+            instance.save()
+            return redirect(event)
+    else:
+        if request.user.is_authenticated():
+            instance, created = Attendance.objects.get_or_create(event=event, attendee=request.user,  # pylint: disable=unused-variable
+                                                                 defaults={'response': 1})
+            form = website_forms.AttendanceForm(instance=instance, initial={'response': instance.response})
```

Comment:
Instead of ```instance.response``` use ```1``` here. User would see ```Yes``` as default value, but it won't be written to the database till the user accepts it.